inc/ocf_mngt.h

```diff
@@ -438,6 +438,14 @@ struct ocf_mngt_cache_attach_config {
 	 *       When used with ocf_mngt_cache_load() it's ignored.
 	 */
 	bool disable_cleaner;
+
+	/**
+	 * @brief Indicates that attach-cache command was called
+	 *
+	 * Used for proper error handling to differentiate between start-cache
+	 * and attach-cache operations, and display proper error message.
+	 */
+	bool is_cmd_attach;
```

Comment:
This is an implementation detail and should not be exposed to the adapter.

Reply:
Sure, I've changed it to utilise cache_state

Reply:
I've moved the differentiation to OCL, but this small change is still required